hibernate-core/src/test/java/org/hibernate/orm/test/bulkid/OracleInlineMutationStrategyIdTest.java

```diff
@@ -1,11 +1,16 @@
 package org.hibernate.orm.test.bulkid;
 
+import org.hibernate.community.dialect.AltibaseDialect;
+
+import org.hibernate.testing.orm.junit.SkipForDialect;
+
 /**
  * Special test that tries to update 1100 rows. Oracle only supports up to 1000 parameters per in-predicate,
  * so we want to test if this scenario works.
  *
  * @author Vlad Mihalcea
  */
+@SkipForDialect(dialectClass = AltibaseDialect.class, reason = "Altibase needs more memory setting to run this test")
```

Comment:
Can't you run your tests with more memory to avoid the skip?

Reply:
I thought tests should be executed in default database settings. So I'm gonna change database setting and if tests runs ok will remove this skip.